various-receipes/data-isolation/data-isolation.test.js

```diff
@@ -104,21 +104,38 @@ describe("/api", () => {
   describe("DELETE /order", () => {
     test("When deleting an existing order, Then should get a successful message", async () => {
       // Arrange
-      const orderToAdd = {
+      const orderToDelete = {
         userId: 1,
         productId: 2,
         externalIdentifier: `id-${getShortUnique()}`, //unique value
       };
       // ️️️✅ Best Practice: Each test acts on its own record. Avoid relying on records from previous tests
       const {
-        data: { id: existingOrderId },
-      } = await axiosAPIClient.post("/order", orderToAdd);
+        data: { id: orderToDeleteId },
+      } = await axiosAPIClient.post("/order", orderToDelete);
+
+      // ✅ Best Practice: Create another order to make sure the delete request deletes only the correct record
+      const anotherOrder = {
+        userId: 1,
+        productId: 2,
+        externalIdentifier: `id-${getShortUnique()}`, //unique value
+      };
+
+      nock("http://localhost/user/").get(`/1`).reply(200, {
+        id: 1,
+        name: "John",
+      });
+      const {
+        data: { id: anotherOrderId },
+      } = await axiosAPIClient.post("/order", anotherOrder);
 
       // Act
-      const receivedResponse = await axiosAPIClient.get(`/order/${existingOrderId}`);
+      const deleteResponse = await axiosAPIClient.delete(`/order/${orderToDeleteId}`);
+      const getOrderResponse = await axiosAPIClient.get(`/order/${anotherOrderId}`);
```

Comment:
should get be part of the Act phase? I think it's more suitable inside assert.

Reply:
This must be after the delete request

Reply:
@mikicho yea, but inside the Assert phase (not that it makes so much of a difference).

Reply:
I did it at first but then set it in the `Act` phase because this is part of what we are testing. that `delete` request deleted only what appropriate records.
WDYT @goldbergyoni ?

Reply:
@mikicho I suggest it should be part of the Assert because, as you said, it's part of the verification and not part of what we Do (Act) to the system. In other words, we don't test /GET, we assume it works, we test that /DELETE didn't overdo